### PR TITLE
WT-7932 Add search terms only if present

### PR DESCRIPTION
**Pull request by sachinmaharjan**
Story: https://jira.blurb.com/browse/WT-7932
Add search terms only if they are present.

### CODE REVIEW

**Comment by sachinmaharjan** [comment]
@pierrel

File: lib/ruby-mws/api/feed.rb [hunk 1]
```diff
@@ -170,9 +170,9 @@ def content_for_product_list(opts={})
                   }
                   xml.MSRP(:currency => entry_hash[:currency]){ xml.text(entry_hash[:standard_price]) }
                   xml.Manufacturer entry_hash[:manufacturer]
-                  entry_hash[:search_terms][:taggings].each do |search_term|
-                    xml.SearchTerms {xml.text(search_term[:tag_name])}
-                  end
+                    entry_hash[:search_terms][:taggings].each do |search_term|
+                      xml.SearchTerms {xml.text(search_term[:tag_name])}
+                    end if !entry_hash[:search_terms].nil?
```
Comment (pierrel) [inline lib/ruby-mws/api/feed.rb]: Can we use `present` here instead? Double-negatives can get confusing.

Comment (pierrel) [inline lib/ruby-mws/api/feed.rb]: Sorry about that I guess `present?` is Rails-only. OK this is good!

**Comment by pierrel** [comment]
+1

**Comment by sachinmaharjan** [comment]
Merging this.

**Pull request merged**